lib/engine.js

```diff
@@ -102,7 +102,10 @@ class Engine {
      * @returns 
      */
     wrapError(sourceError, message, location) {
-        if (!(sourceError instanceof GenericError) && !(sourceError instanceof ClientError)) {
+        if (!(sourceError instanceof GenericError) 
+            && !(sourceError instanceof ClientError)
+            && !(sourceError instanceof RenditionFormatUnsupportedError)
+            && !(sourceError instanceof RenditionTooLarge)) {
```

Comment:
Rendition errors should not be wrapped... Those are useful exceptions we may use to decide to reprocess or not.
(Also they're used in worker tests too).